src/index.js

```diff
@@ -255,6 +243,11 @@ class HCaptcha extends React.Component {
             if (onLoad) onLoad();
         });
       });
+      this.sentryHub?.addBreadcrumb({
+        category: 'react-sdk',
```

Comment:
Is the source not enough info and we need this to understand between loader and react sdc?

Reply:
Thinking this through, I suppose source overwrites the last source. So if we need to track events from loader to sdk it can be tricky, correct?

Reply:
yes correct, currently we set tags again after loader resolves

Reply:
@brdlyptrs good point, this will now be handled in loader